wherehows-web/app/templates/components/dataset-confidential.hbs

```diff
@@ -26,6 +26,73 @@
     </div>
   {{/if}}
 
+  <section class="metadata-prompt">
+    <header class="metadata-prompt__header">
+      <p>
+        This dataset contains the following types of Member data:
+
+        <!--TODO: DSS-6716-->
+        <!-- DRY out with wrapper component that takes the link as an attribute-->
+        <a
+          target="_blank"
+          href="https://iwww.corp.linkedin.com/wiki/cf/display/DWH/List+of+Metadata+for+Data+Sets">
+          <sup>
+            More Info
+
+            <span class="glyphicon glyphicon-question-sign"
+                  title="More information on Dataset classification with examples"></span>
+          </sup>
+        </a>
+      </p>
+    </header>
+  </section>
+
+  <table class="nacho-table nacho-table--bordered nacho-table--stripped">
+    <thead>
+    <tr>
+      <th>Dataset Content Type</th>
+      <th>Is this type of member data contained in this dataset?</th>
+    </tr>
+    </thead>
+    <tbody>
+    {{#each-in datasetClassification as |classification props|}}
+      <tr>
+        <td>
+          {{props.label}}
+
+          <span class="dataset-tag-container">
+            {{#if (eq props.value true)}}
```

Comment:
@justYves nah, the value can be `null` in which case I don't  want to show anything.

Ember's `if` template helper checks for truthiness, but I need a strict equality check in this case